go/mysql/datetime/datetime.go

```diff
@@ -719,6 +719,52 @@ func NewTimeFromStd(t time.Time) Time {
 	}
 }
 
+func NewTimeFromSecondsDecimal(seconds decimal.Decimal) Time {
```

Comment:
Do we need the `Decimal` postfix on the function name here?

Reply:
we usually use `float64` or `int64` for representing `seconds`, so i thought it would be better to add `Decimal` prefix in the func name. altho i think it's already visible from the function signature, so removing it.